runtime/src/lib.rs

```diff
@@ -48,9 +48,19 @@ pub const STABLE_BITCOIN_CONFIRMATIONS: &str = "StableBitcoinConfirmations";
 pub const STABLE_PARACHAIN_CONFIRMATIONS: &str = "StableParachainConfirmations";
 
 // TODO: possibly substitute CurrencyId, VaultId, H256Le
-#[subxt(
-    runtime_metadata_path = "metadata.scale",
-    generated_type_derives = "Debug, Eq, PartialEq, Ord, PartialOrd, Clone"
+#[cfg_attr(
+    feature = "use-parachain-metadata",
+    subxt(
+        runtime_metadata_path = "metadata-parachain.scale",
+        generated_type_derives = "Debug, Eq, PartialEq, Ord, PartialOrd, Clone"
+    )
+)]
+#[cfg_attr(
+    not(feature = "use-parachain-metadata"),
```

Comment:
Tempted to ask for `standalone-metadata` as a separate feature but that's probably not necessary.

Reply:
That'd probably be a bit problematic - the reason why I went with `not(feature = "use-parachain-metadata"),` is because it is easy to enable additional features, but not so easy to disable them. For example, for testing purposes we would require `standalone-metadata`. Ideally, on non-test builds this feature would get disabled, but there is some weirdness with the self-dependency we have in `runtime` that would lead to the feature always getting selected, I believe. I could be wrong, but I don't think it's worth investigating for now